packages/lab/src/tabs/Tab.css

```diff
@@ -1,6 +1,6 @@
 /* class applied to root Tab element */
 .uitkTab {
-  --uitkFocusVisible-inset: 2px 2px 4px 2px;
+  --uitk-focused-outlineInset: 2px 2px 4px 2px;
```

Comment:
are we allowed to override characteristic vars?

Reply:
no, forgot to remove this